src/pybind/mgr/dashboard_v2/README.rst

```diff
@@ -244,3 +244,52 @@ The settings management implementation will make sure that if you change a
 setting value from the python code you will see that change when accessing
 that setting from the CLI and vice-versa.
 
+
+How to listen for manager notifications in a controller?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The manager notifies the modules with several types of cluster events, such
```

Comment:
Maybe better: "...notifies the modules **of** several types of cluster events..."